### PR TITLE
ARTEMIS-4680 - Upgrade the console to use HawtIO 4

### PR DESCRIPTION
**Pull request by andytaylor**
Added a binary distribution

also added binary licenses and license plugin creator

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -208,3 +208,10 @@
 
 The extension is built using Yarn CJS, which is available under an "MIT" license.
 For more information see:  artemis-console-extension/artemis-extension/.yarn/releases/LICENSE-yarn.txt
+
+===============================================================================================================
+= For Yarn License Checker :  artemis-console-extension/artemis-extension/.yarn/plugins/plugin-license.cjs    =
```
[the remark this answers is not in the document]
Comment (andytaylor) [reply LICENSE]: It isnt need to build so have removed the license

File: Releasing.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+## Generating the licenses for the NPM dependencies
+
+Before a release can be the npm license need to be updated with any new dependencies. This file is found under `artemis-console-distribution/src/main/resources/licenses/NPMLicenses.txt`.
```
[the remark this answers is not in the document]
Comment (andytaylor) [reply Releasing.md]: fixed

File: Releasing.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+## Checking out a new empty git repository
+
+Before starting make sure you clone a brand new git as follows as the release plugin will use the upstream for pushing the tags:
+
+```sh
+git clone https://github.com/apache/activemq-artemis-console.git
+cd activemq-artemis-console
+
+git remote add upstream https://gitbox.apache.org/repos/asf/activemq-artemis-console.git
```
[the remark this answers is not in the document]
Comment (andytaylor) [reply Releasing.md]: changed as suggested

File: artemis-console-distribution/src/main/resources/licenses/LICENSE [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+ NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ OF CONTRACT, TORT OR OTHERWISE,  ARISING FROM, OUT OF OR IN CONNECTION
+ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+====================================================================
+= For NPM dependencies                                             =
+====================================================================
+
+every NPM dependency license is found in MPMLicenses.txt
```
[the remark this answers is not in the document]
Comment (andytaylor) [reply artemis-console-distribution/src/main/resources/licenses/LICENSE]: fixed and moved to end

File: Releasing.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
+## Generating the licenses for the NPM dependencies
+
+Before a release can be performed the npm license need to be updated with any new dependencies. This file is found under `artemis-console-distribution/src/main/resources/licenses/NPMLicenses.txt`.
```
Comment (gemmellr) [inline Releasing.md:3]:
```suggestion
Before a release can be performed the npm license need to be updated with any new dependencies. This file is found under `artemis-console-distribution/src/main/resources/licenses/licenses/NPMLicenses.txt`.
```

File: Releasing.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+This is done using the NPM [yarn-plugin-licenses](https://github.com/mhassan1/yarn-plugin-licenses) plugin, you will need to install this if you havent already.
```
Comment (gemmellr) [inline Releasing.md:5]: The .yarnrc file has an entry for this, so is this true at this point?

File: Releasing.md (hunk 5 above)
Comment (andytaylor) [reply Releasing.md]: fixed

File: Releasing.md (hunk 6 above)
Comment (andytaylor) [reply Releasing.md]: removed

**Pull request merged**